wire-ios/Wire-iOS Tests/ConversationMessageCell/ConversationSystemMessageTests.swift

```diff
@@ -310,6 +326,24 @@ final class ConversationSystemMessageTests: ZMSnapshotTestCase {
         verify(message: message)
     }
 
+    func testPotentialGap_addedOneUserAndRemovedMultipleUsers() {
+        let message = MockMessageFactory.systemMessage(with: .potentialGap)!
+
+        message.assignMockAddedUser()
+        message.assignMockRemovedUsers(users: SwiftMockLoader.mockUsers().suffix(4))
+
+        verify(message: message)
+    }
+
+    func testPotentialGap_addedMultipleUsersAndRemovedOneUser() {
```

Comment:
you can already add the test for german too. I guess you can loop through language "en", "de", and set it in the UserDefaults.standard's key `AppleLanguages`

Reply:
@netbe I'm not sure if my German translations are correct. That's why I don't want to add any. I would like for Astrid to take a look first and then add the test.

Reply:
you can setup the test and let Astrid correct the strings on crowdin that's what I meant nothing is blocking you to do that

Reply:
Done!